composer.json

```
@@ -0,0 +1,36 @@
+{
+    "name": "rubaxa/sortable",
+    "type": "component",
+    "description": "Sortable is a JavaScript library for reorderable drag-and-drop lists.",
+    "homepage": "http://rubaxa.github.io/Sortable/",
+    "license": "MIT",
+    "authors": [{
+        "name": "RubaXa",
+        "email": "ibnRubaXa@gmail.com"
+    }],
```

```suggestion
    "authors": [
      {
          "name": "Wayne Van Son",
          "email": "waynevanson@gmail.com"
      },
      {
          "name": "Owen Mills",
          "email" "owen23355@gmail.com"
      }
    ],
```

Although we're not the true authors, we're the individuals maintaining the project.